ovs/lib/albascheduledtask.py

```diff
@@ -0,0 +1,67 @@
+# Copyright 2014 iNuron NV
+#
+# Licensed under the Open vStorage Modified Apache License (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#     http://www.openvstorage.org/license
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+
+"""
+ScheduledTaskController module
+"""
+
+from celery.schedules import crontab
+from ovs.celery_run import celery
+from ovs.dal.lists.albabackendlist import AlbaBackendList
+from ovs.extensions.db.etcd.configuration import EtcdConfiguration
+from ovs.extensions.plugins.albacli import AlbaCLI
+from ovs.lib.helpers.decorators import ensure_single
+from ovs.log.logHandler import LogHandler
+
+logger = LogHandler.get('lib', name='scheduled tasks')
+
+
+class AlbaScheduledTaskController(object):
+    """
+    This controller contains all scheduled task code. These tasks can be
+    executed at certain intervals and should be self-containing
+    """
+
+    job_schedule_x_months = 3
+    job_schedule__x_months_key = '/ovs/alba/backends/job_schedule_x_months'
+    if EtcdConfiguration.exists(job_schedule__x_months_key):
+        job_factor = EtcdConfiguration.get(job_schedule__x_months_key)
```

Comment:
Whats the use of this get?
job_factor value is overwritten in verify_namespaces anyway